src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/util/DesiredOutcomesFactory.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.util
+
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.DesiredOutcome
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceCategory
+import java.util.UUID
+
+class DesiredOutcomesFactory(em: TestEntityManager? = null) : EntityFactory(em) {
```

Comment:
Interesting. I think i have also created one of these in a pr. Happy to go with yours though!